### PR TITLE
Change default Sleep Timers Accuracy setting's value to "Usleep Only"

### LINKED ISSUE #6337: Add timer options to GUI
Followup on https://github.com/RPCS3/rpcs3/issues/6260.
Some games do not like the automatically selected option of "Host" for sleep timings. For this reason, usleep timing should either be default, or the option should be easy to switch in GUI.

### PR DESCRIPTION
**Pull request by elad335**
Bonus: old configs will be updated to use Usleep setting.

Fixes #6337.

### CODE REVIEW

File: rpcs3/Json/tooltips.json [hunk 1]
```diff
@@ -66,7 +66,8 @@
...
 		"strictTextureFlushing": "Forces texture flushing even in situations where it is not necessary/correct. Known to cause visual artifacts, but useful for debugging certain texture cache issues.",
-		"maxSPURSThreads": "Limits the maximum number of SPURS threads in each thread group.\nMay improve performance in some cases, especially on systems with limited number of hardware threads.\nLimiting the number of threads is likely to cause crashes; it's recommended to keep this at default value."
+		"maxSPURSThreads": "Limits the maximum number of SPURS threads in each thread group.\nMay improve performance in some cases, especially on systems with limited number of hardware threads.\nLimiting the number of threads is likely to cause crashes; it's recommended to keep this at default value.",
+		"sleepTimersAccuracy": "Overrider for the default sleep period accuracy of thne operating system.\n Host does not attempt to override the default sleep accuracy, while All Timers attempts to do that at all cases.\n Usleep attempts to do that only for the sleep syscall on PlayStation 3.\n May affect performance unexpectedly."
```
Comment (dio-gh) [inline rpcs3/Json/tooltips.json]: `Overrider for` -> Overrides
`thne` -> the

**Comment by kd-11** [comment]
Idk if debug settings is the best place to put this. Maybe should be in CPU settings? I would have gone with a checkbox instead of a dropdown too. Just something named "Use host sleep timing" but that is just a preference.

File: rpcs3/rpcs3qt/settings_dialog.ui [hunk 2]
```diff
@@ -36,7 +36,7 @@
       </sizepolicy>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>8</number>
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/settings_dialog.ui]: why is everyone commit this LMAO

Comment (elad335) [reply rpcs3/rpcs3qt/settings_dialog.ui]: idk, its something the editor changes automatically or something.

Comment (Megamouse) [reply rpcs3/rpcs3qt/settings_dialog.ui]: I know. just weird to commit it everytime.
It's similar to changing the default value of an unused bool member variable everytime you commit a file lol.

**Comment by StalkerDolg** [comment]
What about option in GUI?

**Comment by MarioSonic2987** [comment]
@StalkerDolg This PR adds that option in GUI on a new tab in settings (Advanced tab).

https://github.com/RPCS3/rpcs3/pull/6350

**Pull request merged**